### DISCUSSION TITLE
Performaning Plastic Deformation Visualization

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#23337] [Q&A Modules: Solid mechanics] [answered]
I am running the sample model "(modules/combined/test/tests/combined_plasticity_temperature/ad_plasticity_temperature_dep_yield.i)". Is there a way to visualize the mesh deformation of the object in relation to the displacements?

**Comment by GiudGiud** [comment, marked as answer]
Hello

There should be a use_displaced-mesh parameter you can use to output the exodus with the displacement.
Otherwise you can also look at the displacement as a field variable

Guillaume

**Reply by TLWise** [reply]
That was a quick and simple fix. Thank you very much for your continued support.
